container/index.js

```diff
@@ -93,26 +93,31 @@ app.get('/tests', function (req, res) {
 // Endpoint to trigger tests called by monitoring app when
 // both FIs are online.
 app.post('/tests', function (req, res) {
-  sendPayment();
+  sendPayment(req.body.assetCode);  
```

Comment:
`/tests` is used to trigger all tests so we shouldn't accept any params here, instead let's do something like:
```
sendPayment("XLM");
sendPayment("TEST");
```